python/pack64.py

```diff
@@ -88,9 +96,14 @@ def unpack64(string, check=True):
     Optionally pass check=False to disable input validation, for circumstances
     where you are sure the input is a properly packed vector.
     '''
-    if check and (len(string) % 3 != 1 or not VALID_CHARS.issuperset(string)):
+    if isinstance(string, bytes):
```

Comment:
These four lines are only relevant if `check` is true.  Can we move them under `if check`?  (That does, of course, necessitate splitting `if check and (stuff):` to `if check: if (stuff):`, but I think that's OK.)

Reply:
...oh wait no I take it back, don't I.  Hold on.

Reply:
Right, yes, sorry, of course the whole point here is that we're using np.frombuffer() now, which means that we need the string to be a bstring.  Forget I said anything. :-)